cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/DRepActivity.hs

```diff
@@ -92,10 +93,10 @@ hprop_check_drep_activity = H.integrationWorkspace "test-activity" $ \tempAbsBas
   gov <- H.createDirectoryIfMissing $ work </> "governance"
 
   -- This proposal should pass
-  let minEpochsToWaitIfChanging = 0 -- The change already provides a min bound
-      minEpochsToWaitIfNotChanging = 3 -- We cannot wait for change since there is no change (we wait a bit)
-      maxEpochsToWaitAfterProposal = 10 -- If it takes more than 10 epochs we give up in any case
-      firstTargetDRepActivity = 3
+  let minEpochsToWaitIfChanging = EpochInterval 0 -- The change already provides a min bound
```

Comment:
What change? I don't know what "The change already provides a min bound" means.

 `activityChangeProposalTest` is missing comments.

Reply:
I just mean that because the value changes, we don't need a minimum wait, as soon as it changes we will know it worked. Maybe I can write that in the comment

Reply:
Fixed in: https://github.com/IntersectMBO/cardano-node/pull/5830/commits/88c6839e7bec22c216379f9fa3b80ab4df46d243